builtin/docker/builder.go

```diff
@@ -133,6 +136,14 @@ build {
 		),
 	)
 
+	doc.SetField(
+		"platform",
+		"set platform if server is multi-platform capable",
+		docs.Summary(
+			"Recommended usage with buildkit enabled.",
```

Comment:
Yup! I made the wording "Must enable Docker buildkit to use the 'platform' flag", since Docker doesn't need to have buidlkit enabled to use the flag, we just require buildkit for it to work with Waypoint.